src/UCTNode.cpp

```diff
@@ -322,15 +362,25 @@ UCTNode* UCTNode::uct_select_child(int color) {
         }
     }
 
+    if (parentvisits == 0) {
+        // sort by score.
+        parentvisits = 1;
+    }
+
     auto numerator = std::sqrt((double)parentvisits);
     auto fpu_reduction = cfg_fpu_reduction * std::sqrt(total_visited_policy);
     // Estimated eval for unknown nodes = original parent NN eval - reduction
     auto fpu_eval = get_net_eval(color) - fpu_reduction;
 
-    for (const auto& child : m_children) {
-        if (!child->valid()) {
-            continue;
-        }
+    auto best_index = 0;
+    auto best_value = -1000.0f;
+
+    FastBoard board;
+    board.reset_board(19);
```

Comment:
What is this doing?

Reply:
I was using this for debug and it didn't get pulled out. Thanks for noticing